docs/src/main/asciidoc/migration-guide-1.x.adoc

```diff
@@ -32,17 +32,10 @@ To use the newly unbundled libraries, add the `spring-cloud-gcp-dependencies` bi
 ----
 <dependencyManagement>
   <dependencies>
-    <dependency>
```

Comment:
Pulling in Spring Cloud BOM is not something we recommend in the main docs, so we should not do it for migration, either.